g05:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  Revolute_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_1
    pid: {p: 50.0, i: 0.0, d: 5}
  Revolute_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_2
    pid: {p: 250.0, i: 30.0, d: 20}
  Revolute_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_3
    pid: {p: 230.0, i: 30.0, d: 15}
  Revolute_4_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_4
    pid: {p: 100.0, i: 10, d: 1}
  Revolute_5_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_5
    pid: {p: 20.0, i: 0.0, d: 0.005}
  Revolute_6_position_controller:
    type: effort_controllers/JointPositionController
    joint: Revolute_7
    pid: {p: 15.0, i: 20.0, d: 0.005}